views/global/markdown.php

```diff
@@ -36,7 +36,7 @@
                 <div class="panel panel-default">
                     <div class="panel-body">
                         <div class="markdown-render">
-                            <?= RichText::output($md) ?>
+                            <?= RichText::output($md, ['fadeIn' => true]) ?>
```

Comment:
@gevorgmansuryan Does this work out of the box or are the RichText Widgets changes neccessary?

Reply:
@luke- no any other changes needed. It was already done in widget